tools/magfit_rotation_gyro.py

```diff
@@ -129,7 +129,7 @@ def magfit(logfile):
                 name = str(m.Name)
                 value = int(m.Value)
             else:
-                name = str(m.param_id)
+                name = m.param_id.decode('ascii')
```

Comment:
Since this is for display, you probably want to also set set errors argument to ignore.